.travis.yml

```diff
@@ -0,0 +1,24 @@
+sudo: false
+
+addons:
+    apt:
+        packages:
+            - doxygen
```

Comment:
For production, we definitely want to use a specific version. The documentation builder should not use a new version without that we manually verify that everything works, otherwise it will break when we don't expect it to.

Either upload the 1.8.12 DEB somewhere or build this version in the script.